### ISSUE TITLE
large batch size support

### ISSUE THREAD

**Issue by xjcvip007** [haon-chen/mmE5#2]
Hello. Since it has been found that GradCache is used in VLM2VEC to enable the model to be trained with a larger batch size, how can we meet the requirements for training with a larger batch size in this project?

**Comment by haon-chen** [comment]
Hi @xjcvip007,
Thank you for your interest in our work! To support training with a larger batch size, we use gradient accumulation (see [this line](https://github.com/haon-chen/mmE5/blob/main/scripts/train/train.sh#L49) in our training script), which allows us to achieve an effective large batch size without requiring excessive GPU memory. 
Let us know if you have any further questions!
